services/explorer/src/explorer/templates/settings/tokens.html

```
@@ -0,0 +1,88 @@
+{% block content %}
+<h2 class="mb-4">API Tokens</h2>
+<p class="text-muted mb-4">
+  Create tokens to connect Claude Desktop, Claude Code, or other MCP clients to your Spotify data.
+</p>
+
+{% if new_token %}
+<div class="alert alert-success alert-dismissible fade show" role="alert">
+  <strong>Token created: {{ new_token_name }}</strong>
+  <p class="mb-2">Copy this token now — it won't be shown again.</p>
+  <div class="input-group mb-2" style="max-width: 600px;">
+    <input type="text" class="form-control font-monospace" value="{{ new_token }}" id="newTokenValue" readonly>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)